src/smart-components/group/group-table-helpers.js

```diff
@@ -49,7 +49,11 @@ export const createRows = (isAdmin, data, _opened, selectedRows = []) =>
           <Fragment key={uuid}>
             <div className="pf-m-inline-flex">
               {isAdmin ? (
-                <Link key={`${uuid}-link`} to={`/groups/detail/${uuid}`}>
+                <Link
+                  key={`${uuid}-link`}
+                  state={{ uuid }}
+                  to={{ pathname: isPlatformDefault ? `/groups/detail/default-access` : `/groups/detail/${uuid}` }}
```

Comment:
I'd change the string in here so we don't repeat the URL
```suggestion
                  to={{ pathname: `/groups/detail/${isPlatformDefault ? 'default-access' : uuid}` }}
```